musicbrainz/PutJSONTogether/app.js

```diff
@@ -1,13 +1,15 @@
 var fs = require("fs");
```

Comment:
Is this script still needed? It is not getting called from /musicbrainz/musicbrainzcli/server.js